pkg/lifecycle/shutdown/shutdown.go

```diff
@@ -68,10 +68,12 @@ func Run() {
 				k8sNodeName = nodeRef
 			}
 
-			// Set node condition, the timeout value is 5 seconds.
-			// Depends on how we configure termination grace period, this operation
-			// may not be successful if it takes too long to update node condition.
-			err := utils.SetNodeNetworkUnavailableCondition(*clientset, k8sNodeName, true, 5*time.Second)
+			hundredYears := 876600 * time.Hour
```

Comment:
🤣 This should be sufficient even for TCP over carrier pigeon networks and the most stubborn of API servers.